activesupport/lib/active_support/cache/file_store.rb

```diff
@@ -118,7 +115,8 @@ def lock_file(file_name, &block) # :nodoc:
         end
 
         # Translate a key into a file path.
-        def key_file_path(key)
+        def normalize_key(key, options)
```

Comment:
While this is private scope, we can't just remove the old method because subclasses can still freely call them.